src/uu/cksum/Cargo.toml

```diff
@@ -16,7 +16,14 @@ path = "src/cksum.rs"
 
 [dependencies]
 clap = { workspace=true }
-uucore = { workspace=true }
+uucore = { version=">=0.0.17", package="uucore", path="../../uucore", features=["sum"] }
+hex = "0.4.3"
+md-5 = "0.10.5"
+sha1 = "0.10.1"
+sha2 = "0.10.2"
+blake2b_simd = "1.0.1"
+blake3 = "1.3.2"
+sm3 = "0.4.1"
```

Comment:
As we moved to workspace,
https://github.com/uutils/coreutils/blob/main/Cargo.toml#L265
thanks to the hard work of @tertsdiepraam 

and we are using most of these crates here:
https://github.com/uutils/coreutils/blob/a75cfd96555f44198b5410f65ec501f89cc6ef30/src/uu/hashsum/Cargo.toml

maybe do the same here?

cc

Reply:
done